### ISSUE TITLE
Sorting feature is not working properly.

### ISSUE THREAD

**Issue by tanujvyas10** [openMF/web-app#481]
**Is your feature request related to a problem? Please describe.**
The tabular titles like Name , Account, External Id, etc. all are having sorting buttons when hover on them, but Only Name's sorting is working and sorting basis on Account, External Id , office address etc. are not working.

**Describe the solution you'd like**
For a table, sorting should be available on the basis of all the column titles.
 
**Describe alternatives you've considered**
Either sorting should be on the basis of all the column titles (Names, Office, Account, etc.) or should be on the basis of  Name only 
 **Screenshot**
![sort5](https://user-images.githubusercontent.com/48439116/68524443-f984f880-02ec-11ea-95b4-29fae3b524e7.jpeg)
![sort4](https://user-images.githubusercontent.com/48439116/68524444-fa1d8f00-02ec-11ea-9feb-45b8b1709050.jpeg)
![sort3](https://user-images.githubusercontent.com/48439116/68524445-fa1d8f00-02ec-11ea-94c8-744a06eaf03f.jpeg)
![sort2](https://user-images.githubusercontent.com/48439116/68524446-fa1d8f00-02ec-11ea-99f7-4856ed58a89a.jpeg)

**I would like to work on it**

**Comment by tanujvyas10** [comment]
@abhaychawla @luckyman20 it this issue approprirate ?

**Comment by abhaychawla** [comment]
@tanujvyas10 It is a valid issue. Although, a decision will be made to majorly revamp this section similar to what was done for the Clients section during this year's GSoC once its code is merged into master.

**Comment by karantakalkar** [comment]
Centers module has been revamped
